### ISSUE TITLE
Finetuning code and 8b Model?

### ISSUE THREAD

**Issue by mehulsuresh** [InternRobotics/VLAC#4]
Hey we tried this model on some of our datasets and it doesn't really have good one shot performance. Are there plans to release the code to fine-tune in the future? Also will the larger 8b model be released?

**Comment by 7-Z-7** [comment]
Thank you for using it. Can you share examples that didn't work well? I might be able to provide some suggestions. The 8b model can currently be experienced at https://vlac.intern-ai.org.cn/, and API access is planned for the future. Fine-tuning related examples are in plans.

**Comment by bbbbbbbbly** [comment]
when the finetune code release? the pi 0.6 all ready use the value function to make RL work.

**Comment by mehulsuresh** [comment]
What pi 0.6 does is more like advantage weighted regression rather than RL.
But yeah everybody is working on this now.
